tests/functional/conftest.py

```diff
@@ -0,0 +1,55 @@
+"""Conftest module for functional tests."""
+import logging
+import os
+import subprocess
+
+import pytest
+import requests
+from requests.adapters import HTTPAdapter
+from urllib3.util.retry import Retry
+
+
+@pytest.fixture(scope="session", autouse=True)
+def install_package():
+    """Install the package to the system and cleanup afterwards.
+
+    Depending on the environment variable TEST_SNAP,
+    it will install the snap or the python package.
+    """
+    software_inventory_exporter_snap = os.environ.get("TEST_SNAP", None)
+    if software_inventory_exporter_snap:
+        # change directory to not import from local modules and force using the snap package
+        cwd = os.getcwd()
+        install_cmd = (
+            f"sudo snap install --dangerous --classic {software_inventory_exporter_snap}".split()
+        )
+        os.chdir("/tmp")
```

Comment:
why we need to change directory to '/tmp' here?

Reply:
I had a issue with `juju-lint` snap (also classic) that if I don't change the working directory, it will run the modules instead of running the snap. Python will have a preference running the code thru the modules and sometimes that can be misleading. E.g: My setup.py was not including non-python files and the functional tests were passing and it wasn't suppose to pass.

Reply:
Seems wired. But if it is true, I suggest to add the comment to explain it.

Reply:
I see because venv is activated and instead of `/snap/bin/juju-lint` it used python version. I that changing the directory to tmp here is maybe not the best approach, but I could not see any other now. Can you put some comments to hat, so if someone will came he/she will know that why it's done. Thanks

Reply:
This is not-blocking

Reply:
Line 21 of this file already has a comment why I'm changing the working directory.